cypress-custom/integration/fee.test.ts

```diff
@@ -66,58 +76,27 @@ describe('Fetch and persist fee', () => {
     _assertFeeFetched(DAI)
   })
 
-  // TODO: not sure if it's easy to test this
+  // TODO:
+  // Difficult to test this as the fee is calculated in the backend
+  // Backend fee calculation is totally unaware/doesn't care what the user's frontend Date time is
+  // meaning any date manipulation here WILL trigger a date change from frontend "/custom/state/fees/updater.tsx",
+  // but backend will not respect that manipulated time when recalculating
+  // see https://github.com/gnosis/gp-v2-services/blob/537ca1856d270b698ca6c7950265198d85c009c7/orderbook/src/api/get_fee_info.rs#L28
```

Comment:
I think this is exactly what you would use the stubbing for instead of relying on real api's response

Reply:
yeah duh fuck im stupid